app/views/facilities/disputed_orders.html.haml

```diff
@@ -0,0 +1,12 @@
+= render "shared/transactions/headers"
+
+- content_for :h1 do
+  = current_facility
+
+- content_for :h2 do
+  %h2= t(".head")
+
+= content_for :top_block do
+  = render "shared/transactions/top", tab: "disputed_orders"
+
+= render "shared/transactions/table" if @order_details.present?
```

Comment:
The `alert` in this partial catches that case https://github.com/tablexi/nucore-open/blob/60971-60973-notifications_menu/app/views/shared/transactions/_top.html.haml#L7 (and I added a translation for it).

But it's still potentially confusing: if your messages menu says you have disputed orders, and you follow the link, there's a chance you won't see any orders listed if the disputed orders are all outside the default search date range.